stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/model/AnnotatedClass.java

```diff
@@ -66,7 +66,7 @@
 
         FieldOption fieldOption = useStag != null ? useStag.value() : null;
         if (fieldOption == null) {
-            useStag = findParentUseStagAnnotation(element);
+            useStag = ElementUtils.findAnnotation(UseStag.class, element);
```

Comment:
Note: This represents a subtle functional change where we consult the type hierarchy rather than the definition location hierarchy to determine the field options.